### DISCUSSION TITLE
Lyrics & hyphenation..

### DISCUSSION THREAD

**Discussion by barrylevingston** [rakuri255/UltraSinger#106] [Ideas]
Hi there, amazing work, I just wanted to share some ideas here: 

-  For lyrics, can we have a step, where the app creates a file after whisper and we can fix the lyrics ourselves?  Of course it would be much better if Ultrasinger downloaded the lyrics & fixed whisper output but until then this can really help... 

- Also for hyphenation, the app should force hyphenate (break the word into pieces or add "~" if the note changes (which the app already knows if it changes during that part)..

**Comment by rakuri255** [comment]
But is there a benefit?
We can already input the txt to repitch it.

**Reply by Kugelblitz78** [reply]
Thant is interesting. How can we input the lyrics to time and repitch it with the vocals only track of a song ?

**Reply by barrylevingston** [reply]
> But is there a benefit? We can already input the txt to repitch it.

I think I found a way to do this.. Fixes most of the problems with transcription errors, but I need help :D   Do you have time to discuss?  @rakuri255

**Reply by rakuri255** [reply]
> Thant is interesting. How can we input the lyrics to time and repitch it with the vocals only track of a song ?

@Kugelblitz78 Sorry for late answare. You can just use the output txt file as input again. So you can fix the lyric and timings after first run with audio and use the txt file for second run

**Reply by rakuri255** [reply]
> > But is there a benefit? We can already input the txt to repitch it.
> 
> I think I found a way to do this.. Fixes most of the problems with transcription errors, but I need help :D Do you have time to discuss? @rakuri255

@barrylevingston Sure. You can reach me on discord. Link is in the ReadMe
